### DISCUSSION TITLE
Remove uvloop

### DISCUSSION THREAD

**Discussion by andersfylling** [fastapi/fastapi#12506] [Questions] [answered]
Is there a way for me to remove the uvloop dependency? It's a cool project, but I'm trying to reduce unecesarry dependencies for our project.

**Comment by YuriiMotov** [comment, marked as answer]
To avoid optional dependencies install FastAPI using command `pip install fastapi` or `poetry add fastapi` (instead of `pip install fastapi[standard]`)

`uvloop` comes with `uvicorn[standard]` which in turn comes when you install FastAPI as `fastapi[standard]` or `fastapi[all]`
